package.json

```diff
@@ -14,7 +14,10 @@
   },
   "dependencies": {
     "@apollo/client": "^3.3.16",
+    "@monsonjeremy/react-leaflet": "^3.2.2",
+    "@react-leaflet/core": ">=1.0.0 <1.1.0 || ^1.1.1",
```

Comment:
Unsually complex version declaration 🤔 I wonder why we can't just use 1.1.1 for instance?

Reply:
I should have removed this one, when I ran into the problem that react-leaflet was not working I tried few different fixes that people suggested. Forgot simplify this one.